lib/main.coffee

```diff
@@ -18,8 +18,8 @@ module.exports =
         text = textEditor.getText()
         parameters = ['--format=json', '-']
         exec = path.join(__dirname, '..', 'node_modules', 'atomlinter-csslint', 'cli.js')
-        cwd = path.dirname(textEditor.getPath())
-        helpers.execNode(exec, parameters, {stdin: text, cwd: cwd}).then (output) ->
+        paths = atom.project.relativizePath(filePath)
+        helpers.execNode(exec, parameters, {stdin: text, cwd: paths.projectPath}).then (output) ->
```

Comment:
How about `Path.dirname(filePath)`, that way the file won't need to be in a project and csslint will also find it's config because presumably like every other linter it keeps searching for config until drive root